app/unopass/pages/_document.tsx

```diff
@@ -3,7 +3,19 @@ import { Html, Head, Main, NextScript } from 'next/document'
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+      <link rel='preconnect' href='https://fonts.googleapis.com' />
+               <link
+                   rel='preconnect'
+                   href='https://fonts.gstatic.com'
+                   crossOrigin='true'
+               />
+               <link
+                   href='https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap'
```

Comment:
you can use next js in built google font implementation

Reply:
Read more here: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts